test_conformance/subgroups/test_workgroup_non_uniform_vote.cpp

```diff
@@ -0,0 +1,344 @@
+//
+// Copyright (c) 2020 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "procs.h"
+#include "subhelpers.h"
+#include "harness/typeWrappers.h"
+
+template <typename Ty, int Which> struct AAN_OPERATION;
+
+template <typename Ty> struct AAN_OPERATION<Ty, 0>
+{
+    static bool calculate(const Ty *mx)
+    {
+        bool taa = false;
+        for (int i = 0; i < NON_UNIFORM; ++i)
+            taa |= mx[i]
+                != 0; // return non zero if value non zero at least for one
```

Comment:
Breaking this expression in two lines is confusing.

Just below, on AAN_OPERANTION<Ty, 1> you wrote the expression in a single line, which is much more clear.